cpp/update_beacon_files.sh

```diff
@@ -38,6 +42,24 @@ else
     error_message+=$'Failed to download the Beacon file from Kalliope.\n'
 fi
 
+# additional beacon files for external references like wikipedia, e.g. ADB/NDB
+declare -a beacon_files=("https://www.historische-kommission-muenchen-editionen.de/beacon_db_register.txt")
```

Comment:
You could use an associative array instead, which contains the URL and the target file name. Then we could also add kalliope and archivportal-d to the same array and keep a single function. I also think that the file name of the 3rd one should be changed, because "beacon_db_register" doesn't tell you on first sight that it comes from adb-ndb or historische-kommission-muenchen-editionen.de or something like that.